gtk/src/toga_gtk/app.py

```diff
@@ -187,6 +189,23 @@ def main_loop(self):
 
         self.loop.run_forever(application=self.native)
 
+    def get_screens(self):
+        display = Gdk.Display.get_default()
+        # CI runs on wayland
+        if os.environ.get("XDG_SESSION_TYPE", "").lower() == "x11":  # pragma: no cover
```

Comment:
There's now an interesting discrepancy between the implementation and the probe - the probe is using `WAYLAND_DISPLAY` as the marker. Any reason for this discrepancy?

(also - the comment says CI runs on Wayland, which it doesn't)

Reply:
Removed it. Thanks.